python/ray/worker.py

```diff
@@ -952,8 +978,8 @@ def _wait_for_and_process_task(self, task):
          ) += 1
 
         reached_max_executions = (self.num_task_executions[task.driver_id().id(
-        )][function_id.id()] == self.function_properties[task.driver_id().id()]
```

Comment:
`task.driver_id()`, `task.task_id()` and `function_id.id()` seem to be used multiple times - can consider abstracting into variable to decrease verbosity.

Reply:
Fixed for `task.driver_id()`. It didn't obviously make sense to me for the others, but I do want to do a subsequent PR cleaning up some of this stuff (but not super sure what the scope of that is yet).